Styles.css

```diff
@@ -0,0 +1,1483 @@
+    user-select: none;
+    font-size: 14px;
+    padding: 0;
+    unicode-bidi: embed;
+    position: relative;
+    overflow: auto;
+    border: 1px solid #ccc
+}
+
+.luna-text-viewer.luna-text-viewer-platform-windows {
+    font-family: "Segoe UI", Tahoma, sans-serif
+}
+
+.luna-text-viewer.luna-text-viewer-platform-linux {
+    font-family: Roboto, Ubuntu, Arial, sans-serif
+}
+
+.luna-text-viewer .luna-text-viewer-hidden,
+.luna-text-viewer.luna-text-viewer-hidden {
+    display: none
+}
+
+.luna-text-viewer .luna-text-viewer-invisible,
+.luna-text-viewer.luna-text-viewer-invisible {
+    visibility: hidden
+}
+
+.luna-text-viewer * {
+    box-sizing: border-box
+}
+
+.luna-text-viewer.luna-text-viewer-theme-dark {
+    color: #d9d9d9;
+    border-color: #3d3d3d;
+    background: #242424
+}
+
+.luna-text-viewer:hover .luna-text-viewer-copy {
+    opacity: 1
+}
+
+.luna-text-viewer-table {
+    display: table
+}
+
+.luna-text-viewer-table .luna-text-viewer-line-number,
+.luna-text-viewer-table .luna-text-viewer-line-text {
+    padding: 0
+}
+
+.luna-text-viewer-table-row {
+    display: table-row
+}
+
+.luna-text-viewer-line-number {
+    display: table-cell;
+    padding: 0 3px 0 8px !important;
+    text-align: right;
+    vertical-align: top;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    border-right: 1px solid #ccc
+}
+
+.luna-text-viewer-line-text {
+    display: table-cell;
+    padding-left: 4px !important;
+    -webkit-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: text;
+    user-select: text
+}
+
+.luna-text-viewer-copy {
+    background: #fff;
+    opacity: 0;
+    position: absolute;
+    right: 5px;
+    top: 5px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    width: 25px;
+    height: 25px;
+    text-align: center;
+    line-height: 25px;
+    cursor: pointer;
+    transition: opacity .3s, top .3s
+}
+
+.luna-text-viewer-copy .luna-text-viewer-icon-check {
+    color: #188037
+}
+
+.luna-text-viewer-text {
+    padding: 4px;
+    font-size: 12px;
+    font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
+    box-sizing: border-box;
+    white-space: pre;
+    display: block
+}
+
+.luna-text-viewer-text.luna-text-viewer-line-numbers {
+    padding: 0
+}
+
+.luna-text-viewer-text.luna-text-viewer-wrap-long-lines {
+    white-space: pre-wrap
+}
+
+.luna-text-viewer-text.luna-text-viewer-wrap-long-lines .luna-text-viewer-line-text {
+    word-break: break-all
+}
+
+.luna-text-viewer-theme-dark {
+    color-scheme: dark
+}
+
+.luna-text-viewer-theme-dark .luna-text-viewer-copy,
+.luna-text-viewer-theme-dark .luna-text-viewer-line-number {
+    border-color: #3d3d3d
+}
+
+.luna-text-viewer-theme-dark .luna-text-viewer-copy .luna-text-viewer-icon-check {
+    color: #81c995
+}
+
+.luna-text-viewer-theme-dark .luna-text-viewer-copy {
+    background-color: #242424
+}

```

Comment:
all the way down to here. that's a lot of code... might not be needed. lmk why you have it?

Reply:
Like i said, thought it was yours. imma just resolve these now